src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/kafka/KafkaExceptionHandler.kt

```diff
@@ -22,10 +22,9 @@ class KafkaExceptionHandler : Thread.UncaughtExceptionHandler, DeserializationEx
 		val message = createMessage("Uncaught exception", e)
 		logger.error(message)
 
-		kafkaPublisher.putMessageOnTopic("null", message)
-
-
 		appConfiguration.state.up = false // Set state.up=false, which (through the Health Endpoint) will trigger a restart of this app instance
```

Comment:
Setter state.up = false før forsøk på å publisere til kafka for å sikre at property settes uavhengig av om exception kastes fra publish  kallet. Kunne alternativt hatt en try block med final og lagt appConfiguration.state.up = false i final blokken